pkg/helm/util.go

```diff
@@ -0,0 +1,23 @@
+package helm
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/loft-sh/log"
+)
+
+func GetBinaryVersion(helmBinaryPath string, log log.Logger) ([]byte, error) {
+	output, err := exec.Command(helmBinaryPath, "version", "--client", "--template", "{{.Version}}").Output()
```

Comment:
Looks like the `--client` has been [ignored / hidden](https://github.com/helm/helm/blob/release-3.0/cmd/helm/version.go#L64-L65) since Helm 3.0, so we could likely drop it entirely.